src/components/TeamCard.js

```diff
@@ -11,10 +11,10 @@ const Team = ({ name, info, photo, role, year }) => {
       containerProps={{ className: "border-0" }}
     >
       <CardMedia>
-        <img src={photo} alt={t("images.teamMemberPhoto")} />
+        <img src={photo} alt={name} />
```

Comment:
Are you sure that there should be something in the `alt` tag? I left mine empty because the photos don't provide any meaning.

Reply:
Good point! I just tried using the screen reader and it was repetitive hearing the name of each team member in the image and subheading below